lib/sync/datasetClientsCleaner.js

```diff
@@ -0,0 +1,78 @@
+var util = require('util');
+var parseDuration = require('parse-duration');
+var async = require('async');
+var _ = require('underscore');
+var DatasetClient = require('./DatasetClient');
+var syncUtil = require('./util');
+
+var storage;
```

Comment:
I'm not that familiar with the prototypical OO stuff in JS, so this probably won't make a lot of sense: Would this make sense as an instance field? I guess not, since `cleanDatasetClients` uses it, and that's a function rather than a method (would that make sense to be a method, or maybe pass `storage` as a parameter to the function?)?